pints/plot.py

```diff
@@ -38,17 +38,17 @@ def function(f, x, lower=None, upper=None, evaluations=20):
     """
     import matplotlib.pyplot as plt
 
-    # Check function get dimension
+    # Check function get n_param
     if not (isinstance(f, pints.LogPDF) or isinstance(f, pints.ErrorMeasure)):
         raise ValueError(
             'Given function must be pints.LogPDF or pints.ErrorMeasure.')
-    dimension = f.n_parameters()
+    n_param = f.n_parameters()
 
     # Check point
     x = pints.vector(x)
-    if len(x) != dimension:
+    if len(x) != n_param:
         raise ValueError(
-            'Given point `x` must have same dimension as function.')
+            'Given point `x` must have same n_param as function.')
```

Comment:
Please call it `n_parameters` in error messages (and function arguments or other public things)

Reply:
Actually, just "number of parameters" in documentation, n_parameters in function arguments
And lots of times below!